DQM/BeamMonitor/plugins/BuildFile.xml

```diff
@@ -9,6 +9,9 @@
 <use name="DataFormats/VertexReco"/>
 <use name="DataFormats/TrackerRecHit2D"/>
 <use name="CondCore/DBOutputService"/>
+<use name="DataFormats/OnlineMetaData"/>
+<use name="dip"/>
+<use name="log4cplus"/>
```

Comment:
@francescobrivio , as these are dependencies of `BeamSpotDipServer` only , so please move these in there e.g

```
<library file="BeamSpotDipServer.cc" name="BeamSpotDipServer">
  <use name="DataFormats/OnlineMetaData"/>
  <use name="dip"/>
  <use name="log4cplus"/>
  <flags EDM_PLUGIN="1"/>
</library>
```

Reply:
OK I will!

Reply:
Fixed in https://github.com/cms-sw/cmssw/pull/35433/commits/d13b53d640eb9f3b86d7ac679b4f3a1aad8a6527
FYI @sikler please do the same for the backport #35428

Reply:
done